OWNERS_ALIASES

```diff
@@ -29,3 +29,20 @@ aliases:
     - ixdy
     - rmmh
     - spxtr
+  sig-node:
```

Comment:
Can we name this `sig-node-reviewers` so it's clear what this list is? I don't want everyone in the sig-node community to feel like they need to be on this list.